bindata/network/ovn-kubernetes/managed/common/004-config.yaml

```diff
@@ -130,6 +130,10 @@ data:
 {{- if (index . "V6JoinSubnet") }}
     v6-join-subnet="{{.V6JoinSubnet}}"
 {{- end }}
+
+    [logging]
```

Comment:
NIT: IIUC the default MaxSize is set to 100 megabytes, can we set that explicitly here so it is clear for us when reading the config itself?